app/views/global_search/_products.html.haml

```diff
@@ -0,0 +1,20 @@
+%h3= Product.model_name.human(count: 2)
+
+%table.table.table-striped.table-hover.js--responsive_table
+  %thead
+    %tr
+      %th= "#{Product.model_name.human} Name"
+      %th= "Product Type"
+      %th= Facility.model_name.human
+      %th= Product.human_attribute_name("description")
+  %tbody
+    - results.each do |product|
+      %tr
+        %td.nowrap.span2
+          = link_to product.name + (product.is_hidden? ? ' (hidden)' : ''), facility_product_path(product.facility, product)
```

Comment:
We need to do some restriction to only show hidden products for members of the facility. And keep in mind, a user might be part of one facility but not another so we need to show the products from one, but not the other.

Reply:
I also wonder if we should show two links for the people who can both purchase _and_ manage the product.

Reply:
I was thinking what was shown would be handled by the ability call in the base searcher:

https://github.com/tablexi/nucore-open/blob/master/app/services/global_search/base.rb#L19

Are there cases that wouldn't be handled by that?

Reply:
You're probably right. I would definitely add a test for it, though.